pkg/scheduler/cache/cache.go

```diff
@@ -770,6 +771,7 @@ func (sc *SchedulerCache) Bind(tasks []*schedulingapi.TaskInfo) error {
 		errTasks, err := sc.Binder.Bind(sc.kubeClient, taskArray)
 		if err == nil {
 			klog.V(3).Infof("bind ok, latency %v", time.Since(tmp))
+			metrics.UpdateTaskBindDuration(metrics.Duration(tmp))
```

Comment:
How about bind failure? Besides, may I know why add this metric? Too many metrics may delay the main scheduling process.

Reply:
When perfromance issuse is reported, the initial thinking to me always would be 'how is the network conditions? is it caused by the laggy network?', but the current implementation only has logs printed, we need to filter-process-analysis the data which is what prometheus prepared for us, right? So with this metric added we could easily figure out the p99/p95 of the binding latency, and easily exlclude/concentrate the network cause from real performance issue.

Reply:
As the above comment we mainly care the p99/p95 latency to clearify the performance issue, failue is not very import to this analysis I thought.